src/systems/a320_systems/src/flaps_computer.rs

```diff
@@ -0,0 +1,1327 @@
+
+        writer.write(
+            "LEFT_SLATS_TARGET_ANGLE",
+            self.slats_demanded_angle.get::<degree>(),
+        );
+        writer.write(
+            "RIGHT_SLATS_TARGET_ANGLE",
+            self.slats_demanded_angle.get::<degree>(),
+        );
+
+        writer.write(
+            "FLAPS_CONF_HANDLE_INDEX_HELPER",
+            self.flaps_conf as u8 as f64,
+        );
+    }
+}
+
+//A placeholder to animate the flaps and slats
+//until implementing Davy's hydraulics
+struct SlatFlapGear {
+    current_angle: Angle,
+    speed: f64,
+    max_angle: Angle,
+    left_position_percent_id: String,
+    right_position_percent_id: String,
+    left_position_angle_id: String,
+    right_position_angle_id: String,
+
+    hyd_green_pressure: f64,
+}
+
+impl SlatFlapGear {
+    const ANGLE_DELTA_DEGREE: f64 = 0.1;
+
+    fn new(speed: f64, max_angle: Angle, surface_type: &str) -> Self {
+        Self {
+            current_angle: Angle::new::<degree>(0.),
+            speed,
+            max_angle,
+            left_position_percent_id: format!("LEFT_{}_POSITION_PERCENT", surface_type),
+            right_position_percent_id: format!("RIGHT_{}_POSITION_PERCENT", surface_type),
+            left_position_angle_id: format!("LEFT_{}_ANGLE", surface_type),
+            right_position_angle_id: format!("RIGHT_{}_ANGLE", surface_type),
+
+            hyd_green_pressure: 0.,
+        }
+    }
+
+    fn update(&mut self, context: &UpdateContext, sfcc_demand: Option<Angle>) {
+        if self.hyd_green_pressure > 2000. {
+            if let Some(demanded_angle) = sfcc_demand {
+                let actual_minus_target: Angle = demanded_angle - self.current_angle;
+                if actual_minus_target.get::<degree>().abs() > Self::ANGLE_DELTA_DEGREE {
+                    self.current_angle += Angle::new::<degree>(
+                        actual_minus_target.get::<degree>().signum()
+                            * self.speed
+                            * context.delta_as_secs_f64(),
+                    );
+                    if self.current_angle < Angle::new::<degree>(0.) {
+                        self.current_angle = Angle::new::<degree>(0.);
+                    }
+                } else {
+                    self.current_angle = demanded_angle;
+                }
+            }
+        }
+    }
+}
+
+impl SimulationElement for SlatFlapGear {
+    fn write(&self, writer: &mut SimulatorWriter) {
+        writer.write(
+            &self.left_position_percent_id,
+            self.current_angle / self.max_angle,
+        );
+        writer.write(
+            &self.right_position_percent_id,
+            self.current_angle / self.max_angle,
+        );
+        writer.write(&self.left_position_angle_id, self.current_angle);
+        writer.write(&self.right_position_angle_id, self.current_angle);
+    }
+
+    fn read(&mut self, reader: &mut SimulatorReader) {
+        self.hyd_green_pressure = reader.read("HYD_GREEN_PRESSURE");
+    }
```

Comment:
I think I already noted this point :). However point of this PR is just to prepare for real integration in hydraulics so maybe it might be "tolerated" to do that. Or maybe just remove the pressure condition is even simpler, even if it will be less fun to see flap moving without hydraulics for some days :p

Reply:
No worries, I already made the appropriate changes. Will push after testing this in-sim again.

Reply:
Suggestion implemented in `28a36b9`. The flap system update is now in the hydraulics module.